Add parser for VHW, correct MWV, improve tests

jaittola opened the pull request:
* Add parsing for the VHW (Water Speed and Heading) statement
 * Fix incorrect units in the parser for MWV (Wind speed & direction)
 * Fix and improve tests for the VHW and MWV statements.

tkurki commented:
Mergeable like this, thanks,a but do you feel like adding schema conformance to the tests? Sort of a separate issue.

See https://github.com/SignalK/n2k-signalk/blob/master/test/129291_set_drift_rapid_update.js#L17

jaittola commented:
I added the tests. Parts of the parser output were not conforming to the schema, and I fixed the cases that were caught by these tests. 

Unfortunately the changes to the tests may be a bit difficult to read in the last commit. Sorry about that.

fabdrol commented:
Looks good, I'd say. +1 for merging, @tkurki you agree?
